docs_user/adoption-attributes.adoc

```diff
@@ -4,27 +4,124 @@
 
 ifeval::["{build}" == "upstream"]
 :OpenShift: OpenShift
-:osp_prev_long: OpenStack
-:osp_acro: OSP
+:rhos_prev_long: OpenStack
+:rhos_acro: OSP
 :OpenStackShort: OSP
 :OpenShiftShort: OCP
-:osp_curr_ver: Antelope
-:osp_prev_ver: Wallaby
+:rhos_curr_ver: Antelope
+:rhos_prev_ver: Wallaby
 :OpenStackPreviousInstaller: TripleO
+
+//Components and services
+
+//Identity service (keystone)
+:identity_service_first_ref: Keystone
+:identity_service: Keystone
+
+//Shared File System service (manila)
 :rhos_component_storage_file_first_ref: Manila
 :rhos_component_storage_file: Manila
+
+//OpenStack Key Manager (barbican)
+:key_manager_first_ref: Barbican
+:key_manager: Barbican
+
+//OpenStack Networking service (neutron)
+:networking_first_ref: Neutron
+:networking: Neutron
+
+//Object Storage service (swift)
+:object_storage_first_ref: Swift
+:object_storage: Swift
+
+//Image service (glance)
+:image_service_first_ref: Glance
+:image_service: Glance
+
+//Compute service (nova)
+:compute_service_first_ref: Nova
+:compute_service: Nova
+
+//Block Storage (cinder)
+:block_storage_first_ref: Cinder
+:block_storage: Cinder
+
+//Dashboard service (horizon)
+:dashboard_first_ref: Horizon
+:dashboard: Horizon
+
+//Bare Metal Provisioning service (ironic)
+:bare_metal_first_ref: Ironic
+:bare_metal: Ironic
+
+//Orchestration service (heat)
+:orchestration_first_ref: Heat
+:orchestration: Heat
+
+//Telemetry service
+:telemetry_first_ref: Telemetry service
+:telemetry: Telemetry service
+
 endif::[]
 
 ifeval::["{build}" == "downstream"]
-:osp_long: Red Hat OpenStack Services on OpenShift
-:osp_acro: RHOSO
-:osp_prev_long: Red Hat OpenStack Platform
+:rhos_long: Red{nbsp}Hat OpenStack Services on OpenShift (RHOSO)
+:rhos_acro: RHOSO
+:rhos_prev_long: Red{nbsp}Hat OpenStack Platform
 :OpenStackShort: RHOSP
-:osp_curr_ver: 18.0
-:osp_prev_ver: 17.1
+:rhos_curr_ver: 18.0
+:rhos_prev_ver: 17.1
 :OpenShift: Red Hat OpenShift Container Platform
 :OpenShiftShort: OpenShift Container Platform
 :OpenStackPreviousInstaller: director
+
+//Components and services
+
+//Identity service (keystone)
+:identity_service_first_ref: Identity service (keystone)
+:identity_service: Identity service
+
+//Shared File System service (manila)
```

Comment:
All good for manila, glance, and swift, thanks. In the comments, maybe keep the s at the end of Systems in Shared File Systems service for the sake of consistency or to avoid copy and paste errors, but that's only a very minor comment on the comments :) In fact, I think Shared File System service could potentially be a better name because it's easier to read than Systems, but that's another story. 

I'm not sure why some entries include OpenStack and others don't, or why the entries for horizon and Telemetry include "Red Hat OpenStack Platform (RHOSP)" when other entries don't and OpenStack isn't a platform any more?

Reply:
You can drop any "OpenStack" prefix from service names.

Reply:
I was going by the conventions that I saw in our downstream docs. For example, on first reference of the Telemetry service,  I saw that it was written as "Red  Red Hat OpenStack Platform (RHOSP) Telemetry service".[1]
But are you saying that I can just use "Telemetry service" on first reference instead?
[1]https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html/deployment_recommendations_for_specific_red_hat_openstack_platform_services/config-recommend-telemetry_config-recommend-telemetry

Reply:
Yes, you can just say Telemetry service.